docs/changes/3.0.0.rst

```diff
@@ -14,6 +14,57 @@ Changes to Supported Platforms
 
 .. towncrier release notes start
 
+
+
+
+Breaking Changes
+----------------
+
+- Starting with Falcon 3.0, Python 2 is no longer supported. Therefore, support
+  for CPython 2.7 and PyPy2.7 was removed in this release. We will continue to
+  maintain the Falcon 2.x series for those who still need to run under Python 2,
+  at least until that platform's scheduled EOL (January 1, 2020 at the time of
+  this writing; see also PEP 373). (27-support)
+- Python 3.4 has now reached its end-of-life per PEP 429, and as a result is no
+  longer supported as of Falcon 3.0. (34-support)
+- The deprecated ``stream_len`` property was removed from the ``Response`` class.
+  Please use ``Response.set_stream()`` or ``Response.content_length`` instead. (stream_len)
+
+
+New & Improved
+--------------
+
+- A new kwarg, ``cors_enable``, was added to the :class:`falcon.API` initializer.
+  ``cors_enable`` can be used to enable a simple blanket CORS policy for all
+  responses. (See also: :ref:`cors`.) (1194-cors)
+- Falcon now supports all WebDAV methods (RFC 2518 and RFC 4918), such as COPY, LOCK, MKCOL, MOVE, PROPFIND, PROPPATCH and UNLOCK. (1426-webdav)
+- The :meth:`~.Response.set_cookie` method now supports setting the
+  `SameSite` cookie attribute. (1557-cookie-samesite)
+- The :meth:`~.Response.add_link` method now supports setting the `crossorigin`
+  link CORS settings attribute. (1574-add-link-crossorigin)
+- The :attr:`~.RequestOptions.auto_parse_form_urlencoded` option is now
+  deprecated in favor of :class:`falcon.media.URLEncodedFormHandler`.
```

Comment:
Again, I can't comment on a file that has just been renamed, but it looks like this deprecation goes with the feature add for `URLEncodedFormHandler`. I was going to ask where it should go, since there is no longer a "deprecations" section. But, now I think the correct move is to combine these fragments.

Reply:
Yeah, as discussed on Gitter, IMHO as long as deprecations are coming from the fact that we introduce newer and improved ways of doing things, it's fine to just mentioned them here.

It's still a bit of an open question what to do with pure deprecations that are not any impromevents _per se_, but we could maybe handle the case if/when it comes.